facade/initial/disabledStatusMetrics.go

```diff
@@ -13,44 +12,42 @@ func NewDisabledStatusMetricsHandler() *disabledStatusMetricsHandler {
 }
 
 // StatusMetricsMapWithoutP2P returns a default response map
-func (d *disabledStatusMetricsHandler) StatusMetricsMapWithoutP2P() map[string]interface{} {
-	return getReturnMap()
+func (d *disabledStatusMetricsHandler) StatusMetricsMapWithoutP2P() (map[string]interface{}, error) {
+	return getReturnValues()
 }
 
 // StatusP2pMetricsMap returns a default response map
-func (d *disabledStatusMetricsHandler) StatusP2pMetricsMap() map[string]interface{} {
-	return getReturnMap()
+func (d *disabledStatusMetricsHandler) StatusP2pMetricsMap() (map[string]interface{}, error) {
+	return getReturnValues()
 }
 
 // StatusMetricsWithoutP2PPrometheusString returns the message that signals that the node is starting
-func (d *disabledStatusMetricsHandler) StatusMetricsWithoutP2PPrometheusString() string {
-	return responseValue
+func (d *disabledStatusMetricsHandler) StatusMetricsWithoutP2PPrometheusString() (string, error) {
+	return "", errNodeStarting
 }
 
 // EconomicsMetrics returns a default response map
-func (d *disabledStatusMetricsHandler) EconomicsMetrics() map[string]interface{} {
-	return getReturnMap()
+func (d *disabledStatusMetricsHandler) EconomicsMetrics() (map[string]interface{}, error) {
+	return getReturnValues()
 }
 
 // ConfigMetrics returns a default response map
-func (d *disabledStatusMetricsHandler) ConfigMetrics() map[string]interface{} {
-	return getReturnMap()
+func (d *disabledStatusMetricsHandler) ConfigMetrics() (map[string]interface{}, error) {
+	return getReturnValues()
 }
 
 //EnableEpochsMetrics returns a default response map
-func (d *disabledStatusMetricsHandler) EnableEpochsMetrics() map[string]interface{} {
-	return getReturnMap()
+func (d *disabledStatusMetricsHandler) EnableEpochsMetrics() (map[string]interface{}, error) {
+	return getReturnValues()
 }
 
 // NetworkMetrics returns a default response map
-func (d *disabledStatusMetricsHandler) NetworkMetrics() map[string]interface{} {
-	return getReturnMap()
+func (d *disabledStatusMetricsHandler) NetworkMetrics() (map[string]interface{}, error) {
+	return getReturnValues()
 }
 
-func getReturnMap() map[string]interface{} {
-	return map[string]interface{}{
-		responseKey: responseValue,
-	}
+func getReturnValues() (map[string]interface{}, error) {
+	return map[string]interface{}{}, errNodeStarting
```

Comment:
```suggestion
	return map[string]interface{}{
		responseKey: responseValue,
	}, errNodeStarting
```
or remove responseValue as it is not used anymore

Reply:
removed